plugin/clang.vim

```diff
@@ -941,16 +953,25 @@ func! s:ClangExecute(root, clang_options, line, col)
     " Please note that '--remote-expr' executes expressions in server, but
     " '--remote-send' only sends keys, which is same as type keys in server...
     " Here occurs a bug if uses '--remote-send', the 'col(".")' is not right.
-    let l:keys = printf('ClangExecuteDone("%s","%s")', l:tmps[0], l:tmps[1])
-    let l:vcmd = printf('%s -s --noplugin --servername %s --remote-expr %s',
-                      \ g:clang_vim_exec, shellescape(v:servername), shellescape(l:keys))
-    let l:command = '('.l:command.';'.l:vcmd.') &'
-    call s:PDebug("s:ClangExecute::cmd", l:command, 2)
-    call system(l:command, l:src)
+    if s:is_windows
+      let l:keys = printf("ClangExecuteDone('%s','%s')", l:tmps[0], l:tmps[1])
+      let l:vcmd = printf('%s -s --noplugin --servername %s --remote-expr %s',
+                        \ g:clang_vim_exec, shellescape(v:servername), shellescape(l:keys))
```

Comment:
Yes, I know. I mean to use single quote version under all platforms, win, linux and OS X.
I've tested on OS X, it works well.